src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
```

Comment:
HTML5 does not support self-closing form tags. You should use a separate closing tag for the form element, like this: `<form action="#" data-qa="big"></form>`.